torch/csrc/Module.cpp

```diff
@@ -624,6 +624,13 @@ static PyObject* initModule() {
 
   ASSERT_TRUE(PyModule_AddObject(module, "has_mkl", at::hasMKL() ? Py_True : Py_False) == 0);
 
+#ifdef _GLIBCXX_USE_CXX11_ABI
```

Comment:
`if defined(_GLIBCXX_USE_CXX11_ABI) && _GLIBCXX_USE_CXX11_ABI != 0`

Reply:
I do guard against that here; if _GLIBCXX_USE_CXX11_ABI is 0 then it evaluates to Py_FAlse
![image](https://user-images.githubusercontent.com/5652049/43474699-9356b368-94c1-11e8-86ab-e9d3fefc2f76.png)
but I could change this to make the logic clearer